### PR TITLE
breaking change: added credential_status_detail in the status assertion

### PR DESCRIPTION
**Pull request by m-basili**
This PR solves #401.

### CODE REVIEW

File: docs/en/pid-eaa-entity-configuration.rst [hunk 1]
```diff
@@ -118,7 +118,12 @@ The *openid_credential_issuer* metadata MUST contain the following claims.
                 
                 - **name**: String value of a display name for the claim.
                 - **locale**: String value that identifies the language of this object represented as a language tag taken from values defined in *BCP47* :rfc:`5646`. There MUST be only one object for each language identifier.
+  * - **credential_status_detail_supported**
+    - JSON object that outlines the details of each validity status supported by the PID/(Q)EAA Provider related to the credentials issued. It contains ``Display`` array containing a list of states with the corresponding descriptions and language identifiers. The parameter that MUST be included are:
```
Comment (peppelinux) [inline docs/en/pid-eaa-entity-configuration.rst]:
```suggestion
    - JSON object that outlines the details of each validity status supported by the PID/(Q)EAA Provider related to the Credentials issued. It contains ``Display`` array containing a list of states with the corresponding descriptions and language identifiers. The parameter that MUST be included are:
```

File: docs/en/pid-eaa-entity-configuration.rst [hunk 2]
```diff
@@ -118,7 +118,12 @@ The *openid_credential_issuer* metadata MUST contain the following claims.
...
 
+        - **state**: String value of a credential status supported.
```
Comment (peppelinux) [inline docs/en/pid-eaa-entity-configuration.rst]:
```suggestion
        - **state**: String value of a Credential status supported.
```

File: docs/en/revocation-lists.rst [hunk 3]
```diff
@@ -648,6 +652,9 @@ When the JWT format is used, the Revocation Assertion MUST contain the following
     * - **credential_status_validity**
       - Boolean value indicating the absolute validity of the Credential linked to the Status Assertion. It MUST be set with the value `false`.
       - `OAUTH-STATUS-ASSERTION`_.
+    * - **credential_status_detail**
+      - Object containing detailed information about the real status of the credential. IT MUST contains ``state`` and related ``description`` claims that MUST be set with one of the values defined in the *credential_status_detail_supported* mapped in the Credential Issuer Metadata.
```
Comment (peppelinux) [inline docs/en/revocation-lists.rst]:
```suggestion
      - Object containing detailed information about the real status of the Credential. It MUST contains ``state`` and related ``description`` claims that MUST be set with one of the values defined in the *credential_status_detail_supported* mapped in the Credential Issuer Metadata.
```

File: docs/en/revocation-lists.rst [hunk 4]
```diff
@@ -699,6 +706,9 @@ When the JWT format is used, the Status Assertion MUST contain the following cla
     * - **credential_status_validity**
       - Boolean value indicating the absolute validity of the Credential linked to the Status Assertion. It is REQUIRED and it MUST be set with the value "false" or "true".
       - `OAUTH-STATUS-ASSERTION`_.
+    * - **credential_status_detail**
+      - REQUIRED if **credential_status_validity** is set to `false`. Object containing detailed information about the real status of the credential. IT MUST contains ``state`` and related ``description`` claims that MUST be set with one of the values defined in the *credential_status_detail_supported* mapped in the Credential Issuer Metadata.
```
Comment (peppelinux) [inline docs/en/revocation-lists.rst]:
```suggestion
      - REQUIRED if **credential_status_validity** is set to `false`. Object containing detailed information about the real status of the Credential. IT MUST contains ``state`` and related ``description`` claims that MUST be set with one of the values defined in the *credential_status_detail_supported* mapped in the Credential Issuer Metadata.
```

File: examples/ec-eaa.json [hunk 5]
```diff
@@ -457,6 +457,20 @@
                     }
                 }
             },
+            "credential_status_detail_supported": {
+                "display": [
+                    {
+                    "state": "Invalida",
+                    "description": "La credenziale non è più utilizzabile in quanto è stata revocata. Questo stato è irreversibile",
```
Comment (peppelinux) [inline examples/ec-eaa.json]:
```suggestion
                    "description": "La Credenziale non è più utilizzabile in quanto è stata revocata. Questo stato è irreversibile",
```

**Review by peppelinux — APPROVED** [review]
@m-basili please accept my editorials.

**Review by grausof — APPROVED** [review]

File: docs/en/revocation-lists.rst [hunk 6]
```diff
@@ -648,6 +652,9 @@ When the JWT format is used, the Revocation Assertion MUST contain the following
     * - **credential_status_validity**
       - Boolean value indicating the absolute validity of the Credential linked to the Status Assertion. It MUST be set with the value `false`.
       - `OAUTH-STATUS-ASSERTION`_.
+    * - **credential_status_detail**
+      - Object containing detailed information about the real status of the Credential. It MUST contains ``state`` and related ``description`` claims that MUST be set with one of the values defined in the *credential_status_detail_supported* mapped in the Credential Issuer Metadata.
```
Comment (peppelinux) [inline docs/en/revocation-lists.rst]:
```suggestion
      - Object containing detailed information about the real status of the Credential. It MUST contains ``state`` and related ``description`` parameters that MUST be set with one of the values defined in the *credential_status_detail_supported* mapped in the Credential Issuer metadata.
```

File: docs/en/revocation-lists.rst [hunk 7]
```diff
@@ -699,6 +706,9 @@ When the JWT format is used, the Status Assertion MUST contain the following cla
     * - **credential_status_validity**
       - Boolean value indicating the absolute validity of the Credential linked to the Status Assertion. It is REQUIRED and it MUST be set with the value "false" or "true".
       - `OAUTH-STATUS-ASSERTION`_.
+    * - **credential_status_detail**
+      - REQUIRED if **credential_status_validity** is set to `false`. Object containing detailed information about the real status of the Credential. IT MUST contains ``state`` and related ``description`` claims that MUST be set with one of the values defined in the *credential_status_detail_supported* mapped in the Credential Issuer Metadata.
```
Comment (peppelinux) [inline docs/en/revocation-lists.rst]:
```suggestion
      - REQUIRED if **credential_status_validity** is set to `false`. Object containing detailed information about the real status of the Credential. IT MUST contains ``state`` and related ``description`` parameters that MUST be set with one of the values defined in the *credential_status_detail_supported* mapped in the Credential Issuer metadata.
```

**Review by peppelinux — CHANGES_REQUESTED** [review]
Any new proposal about ietf oauth status assertion MUST be developed in the related I-D, here:

https://github.com/peppelinux/draft-demarco-oauth-status-assertions

**Review by fmarino-ipzs — APPROVED** [review]

**Review by grausof — APPROVED** [review]

**Comment by m-basili** [comment]
waiting for https://github.com/peppelinux/draft-demarco-oauth-status-assertions/pull/87 approval.

**Review by grausof — APPROVED** [review]

**Review by peppelinux — APPROVED** [review]

**Pull request merged**